packages/core/src/command.ts

```diff
@@ -61,27 +64,42 @@ class Command {
             } else {
               cmdStr += ','
             }
-
+            // safely escape the property - avoid blowing up when attempting to
```

Comment:
it looks like these comments can be deleted now: `safely escape [...]`

now the type is `any` so `for some reason` language not needed anymore